### ISSUE TITLE
You can change the banner

### ISSUE THREAD

**Issue by Multimegamander** [ScarletStudy/DGS1-3DS-Release#35]
use LayeredFS' EXEFS rom hacking, along with the update :p

**Comment by wabberz** [comment]
LayeredFS can only replace:
* the files in the RomFS
* code.bin
* exheader.bin

I'm unaware of it being able to replace the whole ExeFS - please show me a proof of concept.

**Comment by Multimegamander** [comment]
Apache Thunder has a video on YouTube of him rom hacking the Pokemon Omega Ruby & Alpha Sapphire ExeFS
https://www.youtube.com/watch?v=5_Akireswhs

**Comment by wabberz** [comment]
Yes, but he doesn't use LayeredFS for it. We _have_ a translated banner in the ExeFS of the update, but Horizon ignores both the banner and the icon of updates entirely.

It does show up if you rebuild and install the complete game, which kinda defeats the purpose of having a small game update for episodic releases...
